tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
I think this dependency is not allowed, which is why a test fails (search for "fail" in https://ci.tensorflow.org/job/tensorflow-pull-requests-gpu/1833/consoleFull ).

Can you remove this dependency?